op-node/rollup/derive/engine_queue_test.go

```diff
@@ -1192,7 +1192,7 @@ func TestEngineQueue_StepPopOlderUnsafe(t *testing.T) {
 
 	prev := &fakeAttributesQueue{origin: refA}
 
-	ec := NewEngineController(eng, logger, metrics.NoopMetrics, &rollup.Config{}, sync.CLSync)
+	ec := NewEngineController(eng, logger, metrics.NoopMetrics, &rollup.Config{}, sync.CLSync, nil)
```

Comment:
All the tests use the nil daclient arg, it seems not easy to test this feature

Reply:
I think we didn't test every trivial changes or changes with high confidence because of limited engineering resources before, but as more team members join, we add add more test in the future.

Reply:
manual test means test the functionality on the testnet?

Reply:
yes.